doc/geany.txt

```diff
@@ -2208,7 +2208,8 @@ Automatic continuation multi-line comments
       *
 
     on the next line with the correct indentation based on the previous line,
-    as long as the multi-line is not closed by ``*/``.
+    as long as the multi-line is not closed by ``*/``. If the previous line
+    has no ``*`` prefix, the new line will not be changed.
```

Comment:
Suggest "the new line will not be changed" s/b "no ``*`` will be added to the new line" just to be explicit

Reply:
```suggestion
    has no ``*`` prefix, no ``*`` will be added to the new line.
```